lighthouse/classes/event_properties/interfaces.py

```diff
@@ -0,0 +1,225 @@
+from abc import ABC, abstractmethod
```

Comment:
I think it is easier to have smaller files with one concern than larger files with numerous classes etc.

Reply:
Do you mean extracting the exceptions from this file? Or also keeping each class in separate files as well?

Reply:
I'll create 2 modules: exceptions and interfaces and will split interfaces in classes (same as was defintions)

Reply:
Done